src/solvers/cholmod.jl

```diff
@@ -1526,7 +1544,10 @@ end
 function (\)(L::FactorComponent, B::Matrix)
     Matrix(L\Dense(B))
 end
-function (\)(L::FactorComponent, B::SparseVecOrMat)
+function (\)(L::FactorComponent, B::SparseVector)
+    sparsevec(L\Sparse(B))
```

Comment:
I'm a little confuse about why we are returning a sparse vector — isn't the result generally dense?

Reply:
I was confused about that myself, but note that previously we returned a `SparseMatrixCSC`. This change only changes the dimension of the returned array. But, of course, we may as well change to dense return arrays. The actual type of the return value doesn't seem to be tested, so I don't know what "promises" we made.

Reply:
I guess we might as well stay consistent here.

See also https://discourse.julialang.org/t/sparse-solve-with-sparse-rhs/81595/3?u=stevengj